packages/runtime/src/components/builtin/Carousel/Carousel.tsx

```diff
@@ -32,6 +31,7 @@ import { cx } from '@emotion/css'
 import { useResponsiveStyle } from '../../utils/responsive-style'
 import { getBaseBreakpoint } from '../../../state/modules/breakpoints'
 import { useBreakpoints } from '../../../runtimes/react'
+import { ImagesValue } from '@makeswift/prop-controllers'
```

Comment:
Is this type prop controller data or a shared value we compute from the data? These components should mostly be decoupled from controls.

Reply:
I discarded the changes related to the Images. Let's focus on Link for now.